tianshou/trainer/onpolicy.py

```diff
@@ -42,18 +43,20 @@ def onpolicy_trainer(
     :type test_collector: :class:`~tianshou.data.Collector`
     :param int max_epoch: the maximum number of epochs for training. The
         training process might be finished before reaching the ``max_epoch``.
-    :param int step_per_epoch: the number of policy network updates, so-called
-        gradient steps, per epoch.
-    :param int collect_per_step: the number of episodes the collector would
-        collect before the network update. In other words, collect some
-        episodes and do one policy network update.
+    :param int step_per_epoch: the number of environment frames collected per epoch.
```

Comment:
The above `The "step" in trainer means a policy network update.` needs to changed accordingly.

Reply:
done